FESS-GUI/windowsnames.h

```diff
@@ -2,6 +2,12 @@
 #define WINDOWSNAMES
 
 #define MAINWINDOW_TITLE "FlyCAM - Flywheel Control and Monitoring"
+#define MAINWINDOW_ERROR "FlyCAM - Error"
```

Comment:
So, in my opinion it is actually a moot point. None of the strings are reused, so there is no reason to group them together in a header file. See #41